shared/src/main/java/uk/gov/di/authentication/shared/helpers/ValidationHelper.java

```diff
@@ -154,9 +157,13 @@ public static Optional<ErrorResponse> validateVerificationCode(
             return Optional.of(ErrorResponse.ERROR_1002);
         }
 
-        codeStorageService.increaseIncorrectMfaCodeAttemptsCount(emailAddress);
+        if (configurationService.supportAccountCreationTTL()
+                && List.of(VERIFY_EMAIL, VERIFY_PHONE_NUMBER).contains(notificationType)) {
```

Comment:
Sure that phone number should be included here?  Also does VERIFY_EMAIL get used in scenarios other than account creation (account recovery?)